tests/backend/circt/with-guards.futil

```diff
@@ -30,14 +30,15 @@ component main(@go go: 1, @clk clk: 1, @reset reset: 1) -> (@done done: 1) {
   wires {
     group Group1 {
       s0.in = a0.out;
+      m0.addr0 = 1'd1;
       a0.left = m0.read_data;
-      Group1[go] = 1'd0;
+      a0.right = 32'd1;
       c0.in = c0.out;
       Group1[done] = c0.done;
     }
     group Group2 {
       c1.in = (c1.out | c0.done) ? c1.out;
-      Group2[done] = (c1.out & 1'd1 & (c1.out | c0.done)) ? c1.done;
+      Group2[done] = (c1.out & 1'd1 & (c1.out | 1'd1)) ? c1.done;
```

Comment:
This change will cause the Calyx to parse the guard as `c1.out` and not actually let us test the `lower-guards` is working properly. I recommend constructing a complex guard here.

Reply:
Ah good catch. Fixed.